src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -72,7 +72,7 @@ def __init__(self, import_prefix, client_name):
     },
     '2016-00-00-preview': {
         ResourceType.MGMT_STORAGE: '2016-12-01',
-        ResourceType.MGMT_NETWORK: '2016-09-01',
+        ResourceType.MGMT_NETWORK: '2016-12-01',
```

Comment:
The sample profile may be removed in another PR.
https://github.com/Azure/azure-cli/pull/2979/files

No need to change this PR but it just means that API version 2016-12-01 will not be an option.